helm/ffc-ahwr-frontend/values.yaml

```diff
@@ -29,6 +29,7 @@ container:
   cookiePassword: who-likes-cookies-i-like-cookies-everybody-likes-cookies
   eligibilityRequestQueueAddress: ffc-ahwr-eligibility-request
   eligibilityResponseQueueAddress: ffc-ahwr-eligibility-response
+  googleTagManagerKey: GTM-5HKTPLK
```

Comment:
I don't know if the plan is for each environment to use the same tag? If it is, then great and I presume traffic is filtered out within GA.
However, if that isn't the case it might be better to not have a default set which would apply for all deployments and just set the tag in app config for each environment. Then you'd not have a situation where it accidentally uses the default tag when it should have had an env specific one.